source/VirtualScroll/VirtualScroll.test.js

```diff
@@ -115,6 +115,19 @@ describe('VirtualScroll', () => {
       expect(rendered.textContent).toContain('Name 40')
       expect(rendered.textContent).toContain('Name 49')
     })
+
+    it('should scroll to the correct position for :scrollToAlignment "center"', () => {
+      render(getMarkup({
+        scrollToIndex: 99
+      }))
+      const rendered = findDOMNode(render(getMarkup({
+        scrollToAlignment: 'center',
+        scrollToIndex: 49
+      })))
+      // 100 items * 10 item height = 1,000 total item height; 10 items can be visible at a time.
```

Comment:
This comment's a bit misleading I guess, since 11 items are actually visible now (the first and last item are only partially visible).